README.md

```diff
@@ -14,74 +14,90 @@ This repo contains everything you need to work with [Provenance Proof Points](ht
-```
-
-Construct a `ProofPointRegistry` object
 
-```
-const provider = new ethers.providers.JsonRpcProvider();
 const registryRootAddress = EthereumAddress.parse('0x...');
-const registryRoot = new ProofPointRegistryRoot(registryRootAddress, provider);
-const registry = await registryRoot.getRegistry();
-```
-
-### Issue a Proof Point
-
-Each Proof Point has a type, an issuer and some data. The issuer should be a funded Ethereum account that you control. The type should be a type identifying string, for example one of the types documented [here](https://open.provenance.org/developers/specification/). The data can be any javascript object but when serialized to JSON should meet any specification for the type.
-
-```
-// issue a Proof Point
-const result = await registry.issue(
-    'https://open.provenance.org/ontology/ptf/v2/...', // A type identifying URL, such as one from the Provenance ontology
-    '0x...', // The issuer account, a funded account that you control or a did:web identifier that maps to such an account
-    { a: 'b' }, // The data payload of the Proof Point, should match the schema defined by the type
-    '2020-01-01', // Optional valid from date,
-    '2030-01-01', // optional valid until date
+const ipfsSettings = {
+  host: 'example.com',
+  port: 443,
+  protocol: 'https'
+}
+const ethereumProvider = new ethers.providers.JsonRpcProvider();
+
+const proofPointValidator = ProofPointValidator.production(
+    registryRootAddress,
+    ethereumProvider,
+    ipfsSettings
 );
-```
 
-The returned `result` has three fields:
-
-| Field | Notes |
-|-------|-------|
-| `proofPointId` | A unique identifier (ID) for the Proof Point that is also the IPFS address of the Proof Point document |
-| `proofPointObject` | A javascript object that can be serialized to JSON to form the self describing JSON representation of the Proof Point |
-| `transactionHash` | The Ethereum transaction hash of the transaction that issued the Proof Point |
-
-### Validate a Proof Point
+const proofPointId = ProofPointId.parse('Qm...');
 
+const {  
+    isValid,
+    proofPoint,
+    statusCode,
+    statusMessage
+} = await proofPointValidator.validate(proofPointId)
 ```
-// validate a Proof Point object
-const isValid = await registry.validate(proofPointObject);
 
-// validate a Proof Point given its ID
-const isValid = await registry.validateById(proofPointId);
-```
+### Issue or Revoke a Proof Point
+```js
+import { EthereumAddress, EthereumProofPointIssuer } from '@provenance/proof-points';
+import { ethers } from ethers;
+
+const registryRootAddress = EthereumAddress.parse('0x...');
+const ipfsSettings = {
+  host: 'example.com',
+  port: 443,
+  protocol: 'https'
+}
+const ethereumProvider = new ethers.providers.JsonRpcProvider();
+
+const ethereumProofPointIssuer = await EthereumProofPointIssuer.production(
+    registryRootAddress,
+    ipfsSettings,
+    ethereumProvider
+);
 
-### Revoke a Proof Point
+const type = 'https://example.com/1';
```

Comment:
Any URL is valid, so I'm not sure what a better example would be? If I put something specific to provenance like `https://open.provenance.org/ontology/ptf/v2/CertificationCredential` it might imply that that is what you need to put.